orchestrate.py

```diff
@@ -232,7 +233,7 @@ def main():
         (r"/spawn/?(/user/\w+(?:/.*)?)?", SpawnHandler),
         (r"/api/spawn/", APISpawnHandler),
         (r"/(user/\w+)(?:/.*)?", LoadingHandler),
-        (r"/stats", StatsHandler),
```

Comment:
The new human page can be at `/statistics` or `/info` rather than re-using the URL.